src/vinylcontrol/vinylcontrol.cpp

```diff
@@ -21,8 +22,8 @@ VinylControl::VinylControl(ConfigObject<ConfigValue> * pConfig, QString group)
     trackSamples        = new ControlObjectThread(group, "track_samples");
     trackSampleRate     = new ControlObjectThread(group, "track_samplerate");
     vinylSeek           = new ControlObjectThread(group, "vinylcontrol_seek");
-    controlScratch      = new ControlObjectThread(group, "vinylcontrol_rate");
-    rateSlider          = new ControlObjectThread(group, "rate");    //Range -1.0 to 1.0
+    m_pVCRate = new ControlObjectSlave(group, "vinylcontrol_rate");
```

Comment:
The old names and comments are just wrong. I have tried only fixed the issues related to this PR. 
Do you plan to fix it an other way round later? You are still able to do, but it feels stupid, to fix an issue here, revert it and fix it again later. If you can effort the time to remove my changes, you are free to do. Just do a PR against this branch or unfix it in master.